src/components/about-us/section-03/content.js

```diff
@@ -163,71 +177,156 @@ const MoreInfo = styled.div`
   }
 `
 
-const YearTag = styled.p`
-  position: absolute;
-  left: 0;
-  top: 0;
-  font-family: ${font.family.english.roboto}, ${font.family.sansSerifFallback};
-  font-weight: ${font.weight.bold};
-`
-
-const YearTagOnDesktopAbove = YearTag.extend`
+const PageWrapper = styled.div`
+  position: relative;
+  width: 100%;
   ${screen.desktopAbove`
-    transform: translateX(-100%);
-    color: ${colors.white};
-    background: ${colors.black};
-    padding: 2px;
-  `}
-  ${screen.overDesktop`
-    font-size: 15px;
-  `}
-  ${screen.desktop`
-    font-size: 13px;  
-  `}
-  ${screen.tabletBelow`
-    display: none;
+    padding: 40px 46px;
+    height: 100%;
+    overflow-y: hidden;
   `}
 `
 
-const YearTagOnTabletBelow = YearTag.extend`
+const RecordsInaYear = styled.ul`
+  width: 100%;
+  transform-style: preserve-3d;
+  margin-top: 2px;
+  margin-bottom: ${props => props.unfold ? '9px' : '0'};
+  padding: 0;
+  list-style: none;
+`
+
+const Record = styled.li`
+  width: 100%;
+  margin: auto 0;
+  color: ${colors.black};
   position: relative;
+  min-height: 204px;
+  height: ${props => props.unfold ? 'auto' : '204px'};
+  transition: all ${transitionDuration} linear;
+  border-bottom: solid 1px ${borderBottomColor};
   text-align: left;
+  padding: 20px 21px 19px 15px;
+  background: ${colors.gray.gray96};
+  a{
+    color: ${colors.black};
+  }
+  &:nth-child(odd) {
+    transform-origin: 50% 0 0;
+    transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(-90deg)'};
+    will-change: transform;
+    &:before {
+      content: '';
+      background-image: linear-gradient(to top, ${colors.gray.gray64}, ${colors.gray.gray96} 80%);
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: ${props => props.unfold ? '0' : '100%'};
+      z-index: calc(${defaultZIndex} - 1);
+      transition: height .25s;      
+    }
+  }
+  &:nth-child(even) {
+    transform-origin: 50% 100% 0;
+    transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(90deg)'};
+    margin-top: ${props => props.unfold ? '0' : 'calc(-204px * 2)'};
+    will-change: transform, margin-top;
+    &:before {
+      content: '';
+      background-image: linear-gradient(to bottom, ${colors.gray.gray64}, ${colors.gray.gray96} 80%);
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: ${props => props.unfold ? '0' : '100%'};
+      z-index: calc(${defaultZIndex} - 1);
+      transition: height .25s;      
+    }
+  }
```

Comment:
Could we reuse the unfold/fold styled components both in section-03 and section-05 ?